core/cmd/app.go

```diff
@@ -729,6 +756,24 @@ func NewApp(client *Client) *cli.App {
 					Name:   "create",
 					Usage:  "Create a new node",
 					Action: client.CreateNode,
+					Flags: []cli.Flag{
+						cli.StringFlag{
+							Name:  "name",
+							Usage: "node name",
+						},
+						cli.StringFlag{
+							Name:  "ws-url",
+							Usage: "Websocket URL",
+						},
+						cli.StringFlag{
+							Name:  "ws-url",
```

Comment:
Is this supposed to be `http-url`?

Surprised this doesnt throw some kind of error at runtime